packages/core/helper-plugin/lib/src/components/Form/index.js

```diff
@@ -1,27 +1,27 @@
-import React, { useEffect } from 'react';
-import { Form, useFormikContext, getIn } from 'formik';
+import React, { useEffect, useRef } from 'react';
+import { Form, useFormikContext } from 'formik';
 
 const FormWithFocus = props => {
+  const formRef = useRef(null);
   const { isSubmitting, isValidating, errors, touched } = useFormikContext();
 
   useEffect(() => {
     if (isSubmitting && !isValidating) {
-      const errorNames = Object.keys(touched).filter(error => getIn(errors, error));
-
-      if (errorNames.length) {
-        let errorEl;
-
-        errorNames.forEach(errorKey => {
-          const selector = `[name="${errorKey}"]`;
-
-          if (!errorEl) {
-            errorEl = document.querySelector(selector);
-          }
-        });
-
-        errorEl.focus();
+      const errorsInForm = formRef.current.querySelectorAll('[data-strapi-field-error]');
```

Comment:
ℹ️ moving to the new data attribute in order to target elements like "buttons" in select